src/ContentRepository/Schema/UnknownContentHandler.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using SenseNet.ContentRepository.Schema;
+using SenseNet.ContentRepository.Storage;
+
+namespace SenseNet.Storage
+{
+    [ContentHandler]
+    public class UnknownContentHandler : Node
+    {
+        public UnknownContentHandler(Node parent) : this(parent, null) { }
+        public UnknownContentHandler(Node parent, string nodeTypeName) : base(parent, nodeTypeName) { }
+        protected UnknownContentHandler(NodeToken nt) : base(nt) { }
+        public override bool IsContentType { get; } = false;
+
+        public override void Save(NodeSaveSettings settings)
+        {
+            ThrowException("save");
+        }
+        public override void CopyTo(Node target)
+        {
+            ThrowException("copy");
+        }
+        public override void CopyTo(Node target, string newName)
+        {
+            ThrowException("copy");
+        }
+        public override void Delete()
+        {
+            ThrowException("delete");
+        }
+        public override void FinalizeContent()
+        {
+            ThrowException("finalize");
+        }
+        public override void ForceDelete()
+        {
+            ThrowException("delete");
+        }
+        public override void MoveTo(Node target)
+        {
+            ThrowException("move");
+        }
+
+        private static void ThrowException(string operation)
+        {
+            throw new InvalidOperationException($"Cannot {operation} a content with an unknown handler.");
```

Comment:
This exception type doesn't describe the problem. I think `NotSupportedException` should a better choice. `SnNotSupportedException` also can be good if the message won't contain "in this version" term.
Note: the specification of the `InvalidOperationException` says:
"The exception that is thrown when a method call is invalid for the object's current state."